### ISSUE TITLE
The spacing between some characters is too large

### ISSUE THREAD

**Issue by huyinjie** [iissnan/hexo-theme-next#2107]
### I agree and want to create new issue

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
![image](https://user-images.githubusercontent.com/8533957/36548666-da35d5e2-182b-11e8-925b-db2a3ae4a84a.png)

### Actual behavior 
![image](https://user-images.githubusercontent.com/8533957/36548623-c1c2628c-182b-11e8-9101-7b7f84af1152.png)

While I open it in my Browser with full screen it get error but when I zoom out the window it's OK.
The picture mentioned above is  [here](https://huyinjie.xyz/2018/02/22/%E4%BD%BF%E7%94%A8Telegram-cli%E5%AE%8C%E6%88%90%E8%87%AA%E5%8A%A8%E5%8C%96%E7%AD%BE%E5%88%B0/).

### Hexo Information
```
3.5.0
```

### NexT Information

**NexT Version:**
```
[x] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[x] Mist
[] Pisces
[] Gemini
```

**Comment by sli1989** [comment]
fixed in [#24](https://github.com/theme-next/hexo-theme-next/issues/24)

**Comment by huyinjie** [comment]
It works,thanks
